doc/init.md

```diff
@@ -20,9 +20,9 @@ The macOS configuration assumes bitcoind will be set up for the current user.
 Configuration
 ---------------------------------
 
-At a bare minimum, bitcoind requires that the rpcpassword setting be set
-when running as a daemon.  If the configuration file does not exist or this
-setting is not set, bitcoind will shut down promptly after startup.
+Running bitcoind as a daamon does not require any manual configuration. You may
+set the rpcpassword setting in the `bitcoin.conf` configuration file to override
```

Comment:
nit: I personally prefer "overwrite"

Reply:
technically this is an "override" because it is changing a default behavior and not replacing(overwrite) something that is existing.

Also override terminology is being used in other places -> https://github.com/bitcoin/bitcoin/blob/9546a785953b7f61a3a50e2175283cbf30bc2151/doc/bitcoin-conf.md

so to be consistent, guess it is okay to keep it "override".